### ISSUE TITLE
Code Snippet in Content Section

### ISSUE THREAD

**Issue by isaac2004** [joeaudette/cloudscribe.SimpleContent#156]
Hello,

I have been poking around the blog for a few hours, and the concept is great, and I am considering using it over something like WP or whatever. What I would like to see is the ability to paste pretty code into the Content Editor. I know CKEditor has this
http://ckeditor.com/addon/codesnippet

Is this something that can be added easy to the Web Project?

**Comment by joeaudette** [comment]
Hi,

Great suggestion! I decided to include that plugin by default and just pushed new nugets that include it. There may be other things that people would like to customize in the editor so I also just created a wiki page to explain [Customizing the Editor](https://github.com/joeaudette/cloudscribe.SimpleContent/wiki/Customizing-The-Editor). I included notes there about the codesnippet plugin because you do still need to add some css and js to do the highlighting outside the editor and I didn't want to opt everyone into that by default.
